source/ingress.go

```diff
@@ -65,6 +73,28 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 		return endpoints
 	}
 
+	if len(ing.Spec.Rules) == 0 && fqdntemplate != "" {
+		tmpl, err := template.New("endpoint").Funcs(template.FuncMap{
```

Comment:
Yep, this is exactly what I've wanted to implement ( #143 ). If rules are set in the ingress use the hosts from there, but if not; so only an ingress with a default backend; use the template to generate a DNS record for this so it does not end up without any records.